### PR TITLE
swapped references of /auth/login to https://theskills.com/sign-in an…

### PR DESCRIPTION
**Pull request by bashby44**
- swapped references of /auth/login to https://theskills.com/sign-in
- added text and button to sign in when query.error.message=UNAUTHORIZED instead of just displaying UNAUTHORIZED

### CODE REVIEW

File: components/Shell.tsx [hunk 1]
```diff
@@ -53,7 +53,7 @@ function useRedirectToLoginIfUnauthenticated() {
   useEffect(() => {
     if (!loading && !session) {
       router.replace({
-        pathname: "/auth/login",
+        pathname: "https://theskills.com/sign-in",
```
Comment (mcasey8540) [inline components/Shell.tsx]: To streamline dev and staging testing, can you change this to relative url `${process.env.THETIS_SITE_HOST}/sign-in}`

File: lib/QueryCell.tsx [hunk 2]
```diff
@@ -57,11 +61,22 @@ export function QueryCell<TData, TError extends ErrorLike>(
...
-    return (
-      opts.error?.(query) ?? (
-        <Alert severity="error" title="Something went wrong" message={query.error.message} />
-      )
-    );
+    if (query.error.message === "UNAUTHORIZED") {
+      return (
+        <>
+          <p className="my-4 text-sm text-gray-500">{t("youve_been_logged_out")}</p>
+          <Button color="primary" href={"https://theskills.com/sign-in"}>
```
Comment (mcasey8540) [inline lib/QueryCell.tsx]: `${process.env.THETIS_SITE_HOST}/sign-in}`

File: pages/api/auth/[...nextauth].tsx [hunk 3]
```diff
@@ -97,7 +97,7 @@ export default NextAuth({
   },
   debug: process.env.NODE_ENV !== "production",
   pages: {
-    signIn: "/auth/login",
+    signIn: "https://theskills.com/sign-in",
```
Comment (mcasey8540) [inline pages/api/auth/[...nextauth].tsx]: `${process.env.THETIS_SITE_HOST}/sign-in}`

File: pages/auth/error.tsx [hunk 4]
```diff
@@ -40,7 +40,7 @@ export default function Error() {
             </div>
           </div>
           <div className="mt-5 sm:mt-6">
-            <Link href="/auth/login">
+            <Link href="https://theskills.com/sign-in">
```
Comment (mcasey8540) [inline pages/auth/error.tsx]: `${process.env.THETIS_SITE_HOST}/sign-in}`

File: pages/auth/forgot-password/[id].tsx [hunk 5]
```diff
@@ -84,7 +84,7 @@ export default function Page({ resetPasswordRequest, csrfToken }: Props) {
             </h2>
           </div>
           <p>{t("password_has_been_reset_login")}</p>
-          <Link href="/auth/login">
+          <Link href="https://theskills.com/sign-in">
```
Comment (mcasey8540) [inline pages/auth/forgot-password/[id].tsx]: `${process.env.THETIS_SITE_HOST}/sign-in}`

File: pages/auth/forgot-password/index.tsx [hunk 6]
```diff
@@ -142,7 +142,7 @@ export default function ForgotPassword({ csrfToken }: { csrfToken: string }) {
                   </button>
                 </div>
                 <div className="space-y-2">
-                  <Link href="/auth/login">
+                  <Link href="https://theskills.com/sign-in">
```
Comment (mcasey8540) [inline pages/auth/forgot-password/index.tsx]: `${process.env.THETIS_SITE_HOST}/sign-in}`

File: pages/auth/logout.tsx [hunk 7]
```diff
@@ -37,7 +37,7 @@ export default function Logout() {
             </div>
           </div>
           <div className="mt-5 sm:mt-6">
-            <Link href="/auth/login">
+            <Link href="https://theskills.com/sign-in">
```
Comment (mcasey8540) [inline pages/auth/logout.tsx]: `${process.env.THETIS_SITE_HOST}/sign-in}`

File: pages/auth/signup.tsx [hunk 8]
```diff
@@ -174,7 +174,7 @@ export const getServerSideProps = async (ctx: GetServerSidePropsContext) => {
     return {
       redirect: {
         permanent: false,
-        destination: "/auth/login?callbackUrl=" + ctx.query.callbackUrl,
+        destination: "https://theskills.com/sign-in",
```
Comment (mcasey8540) [inline pages/auth/signup.tsx]: `${process.env.THETIS_SITE_HOST}/sign-in}`

File: pages/bookings/[status].tsx [hunk 9]
```diff
@@ -42,15 +42,30 @@ export default function Bookings() {
 
   const isEmpty = !query.data?.pages[0]?.bookings.length;
 
+  const statusError = (query: { status?: string; error?: { message?: string } | null }) => {
+    if (query?.status === "error") {
+      if (query?.error?.message === "UNAUTHORIZED") {
+        return (
+          <>
+            <p className="my-4 text-sm text-gray-500">{t("youve_been_logged_out")}</p>
+            <Button color="primary" href={"https://theskills.com/sign-in"}>
```
Comment (mcasey8540) [inline pages/bookings/[status].tsx]: `${process.env.THETIS_SITE_HOST}/sign-in}`

File: pages/bookings/index.tsx [hunk 10]
```diff
@@ -7,7 +7,7 @@ function RedirectPage() {
 export async function getServerSideProps(context) {
   const session = await getSession(context);
   if (!session?.user?.id) {
-    return { redirect: { permanent: false, destination: "/auth/login" } };
+    return { redirect: { permanent: false, destination: "https://theskills.com/sign-in" } };
```
Comment (mcasey8540) [inline pages/bookings/index.tsx]: `${process.env.THETIS_SITE_HOST}/sign-in}`

File: pages/event-types/[type].tsx [hunk 11]
```diff
@@ -1233,7 +1233,7 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
     return {
       redirect: {
         permanent: false,
-        destination: "/auth/login",
+        destination: "https://theskills.com/sign-in",
```
Comment (mcasey8540) [inline pages/event-types/[type].tsx]: `${process.env.THETIS_SITE_HOST}/sign-in}`

File: pages/getting-started.tsx [hunk 12]
```diff
@@ -509,7 +509,7 @@ export async function getServerSideProps(context: NextPageContext) {
     return {
       redirect: {
         permanent: false,
-        destination: "/auth/login",
+        destination: "https://theskills.com/sign-in",
```
Comment (mcasey8540) [inline pages/getting-started.tsx]: `${process.env.THETIS_SITE_HOST}/sign-in}`

File: pages/index.tsx [hunk 13]
```diff
@@ -7,7 +7,7 @@ function RedirectPage() {
 export async function getServerSideProps(context) {
   const session = await getSession(context);
   if (!session?.user?.id) {
-    return { redirect: { permanent: false, destination: "/auth/login" } };
+    return { redirect: { permanent: false, destination: "https://theskills.com/sign-in" } };
```
Comment (mcasey8540) [inline pages/index.tsx]: `${process.env.THETIS_SITE_HOST}/sign-in}`

File: pages/settings/profile.tsx [hunk 14]
```diff
@@ -455,7 +455,7 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
   const session = await getSession(context);
 
   if (!session?.user?.id) {
-    return { redirect: { permanent: false, destination: "/auth/login" } };
+    return { redirect: { permanent: false, destination: "https://theskills.com/sign-in" } };
```
Comment (mcasey8540) [inline pages/settings/profile.tsx]: `${process.env.THETIS_SITE_HOST}/sign-in}`

**Review by mcasey8540 — CHANGES_REQUESTED** [review]

File: components/Shell.tsx (hunk 1 above)
Comment (bashby44) [reply components/Shell.tsx]: Yeah good call

**Review by mcasey8540 — APPROVED** [review]

**Pull request merged**